src/zenml/zen_stores/schemas/pipeline_schemas.py

```diff
@@ -138,6 +138,7 @@ class PipelineRunSchema(SQLModel, table=True):
     )
     pipeline: PipelineSchema = Relationship(back_populates="runs")
 
+    status: ExecutionStatus
```

Comment:
Could you please make this field optional to maintain backwards compatibility with existing database entries ? Until we implement a mechanism for database migration, this should be our strategy, otherwise users will have to drop their DBs when they update zenml, which is not desirable. I'm getting this if I use my current database with this PR: 
```OperationalError: (sqlite3.OperationalError) no such column: pipelinerunschema.status
[SQL: SELECT pipelinerunschema.id 
FROM pipelinerunschema 
WHERE pipelinerunschema.status = ?]
[parameters: (<ExecutionStatus.RUNNING: 'running'>,)]```

Reply:
On second thought, the solution I suggested won't work, because the sqlmodel schema will still want to access that new field on reads/writes.

Reply:
Perhaps it's time to introduce SQLModel to alembic: https://github.com/tiangolo/sqlmodel/issues/85